specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2021-10-01-preview/AutomationRules.json

```diff
@@ -545,367 +533,651 @@
-          "description": "The type of event the automation rule triggers on",
-          "enum": [
-            "Created"
-          ],
-          "type": "string",
-          "x-ms-enum": {
-            "modelAsString": true,
-            "name": "TriggersWhen",
-            "values": [
-              {
-                "description": "Trigger on created objects",
-                "value": "Created"
-              }
-            ]
-          }
+          "$ref": "#/definitions/triggersWhen"
         },
         "conditions": {
           "description": "The conditions to evaluate to determine if the automation rule should be triggered on a given object",
+          "type": "array",
           "items": {
             "$ref": "#/definitions/AutomationRuleCondition"
+          }
+        }
+      }
+    },
+    "ConditionType": {
+      "enum": [
+        "Property"
+      ],
+      "type": "string",
+      "example": "Property",
+      "x-ms-enum": {
+        "name": "ConditionType",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Property",
+            "description": "Evaluate an object property value"
+          }
+        ]
+      }
+    },
+    "IncidentClassification": {
+      "enum": [
+        "Undetermined",
+        "TruePositive",
+        "BenignPositive",
+        "FalsePositive"
+      ],
+      "type": "string",
+      "example": "Undetermined",
+      "x-ms-enum": {
+        "name": "IncidentClassification",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Undetermined"
+          },
+          {
+            "value": "TruePositive"
+          },
+          {
+            "value": "BenignPositive"
+          },
+          {
+            "value": "FalsePositive"
+          }
+        ]
+      }
+    },
+    "IncidentClassificationReason": {
+      "enum": [
+        "SuspiciousActivity",
+        "SuspiciousButExpected",
+        "IncorrectAlertLogic",
+        "InaccurateData"
+      ],
+      "type": "string",
+      "example": "SuspiciousActivity",
+      "x-ms-enum": {
+        "name": "IncidentClassificationReason",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "SuspiciousActivity"
+          },
+          {
+            "value": "SuspiciousButExpected"
+          },
+          {
+            "value": "IncorrectAlertLogic"
+          },
+          {
+            "value": "InaccurateData"
+          }
+        ]
+      }
+    },
+    "IncidentOwnerType": {
+      "enum": [
+        "Unknown",
+        "User",
+        "Group"
+      ],
+      "type": "string",
+      "example": "Unknown",
+      "x-ms-enum": {
+        "name": "IncidentOwnerType",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Unknown"
+          },
+          {
+            "value": "User"
+          },
+          {
+            "value": "Group"
+          }
+        ]
+      }
+    },
```

Comment:
shouldn't all these incident models (classification, classification reason, owner type) be in the common incident models file?